dd-sdk-android/build.gradle.kts

```diff
@@ -150,6 +150,8 @@ unMock {
     keepStartingWith("com.android.internal.util.")
     keepStartingWith("android.util.")
     keep("android.content.ComponentName")
+    keep("android.os.Looper")
```

Comment:
Why do you need this ?

Reply:
Because the tests are using the Loopers.